carma_wm_ctrl/test/WMBroadcasterTest.cpp

```diff
@@ -569,19 +572,20 @@ TEST(WMBroadcaster, addAndRemoveGeofence)
   gf_msg.geometry.nodes.push_back(pt);
   gf_ptr->gf_pts = wmb.getPointsInLocalFrame(gf_msg);
   gf_ptr->affected_parts_ = wmb.getAffectedLaneletOrAreas(gf_ptr->gf_pts);
+  gf_ptr->msg_ = gf_msg;
 
-  ASSERT_EQ(gf_ptr->affected_parts_.size(), 2);
-  ASSERT_EQ(gf_ptr->affected_parts_[1].id(), 10000);
-  ASSERT_EQ(gf_ptr->affected_parts_[1].regulatoryElements()[0]->id(), old_speed_limit->id()); // old speed limit
-  ASSERT_EQ(gf_ptr->affected_parts_[1].regulatoryElements().size(), 4); // old speed limit and other map conforming regulations
+  ASSERT_EQ(gf_ptr->affected_parts_.size(), 3);
+  ASSERT_EQ(gf_ptr->affected_parts_[0].id(), 10000);
+  ASSERT_EQ(gf_ptr->affected_parts_[0].regulatoryElements()[0]->id(), old_speed_limit->id()); // old speed limit
+  ASSERT_EQ(gf_ptr->affected_parts_[0].regulatoryElements().size(), 4); // old speed limit and other map conforming regulations
   // process the geofence and change the map
 
   wmb.addGeofence(gf_ptr);
   ASSERT_EQ(map_update_call_count, 1);
   /*Analyze prev_regems_.size()*/
-  ASSERT_EQ(gf_ptr->prev_regems_.size(), 2);
-  ASSERT_EQ(gf_ptr->prev_regems_[0].first, 10007);
-  ASSERT_EQ(gf_ptr->prev_regems_[1].second->id(), old_speed_limit->id());
+  ASSERT_EQ(gf_ptr->prev_regems_.size(), 3);
+  ASSERT_EQ(gf_ptr->prev_regems_[0].first, 10000);
+  ASSERT_EQ(gf_ptr->prev_regems_[0].second->id(), old_speed_limit->id());
```

Comment:
Please confirm that all these unit test fixes make conceptual sense, not just updating to what the system is outputting as such a failure could indicate a bug.

Reply:
yeah I confirmed this is correct as this is the first lanelet of the three lanelets detected 10000, 10007, 10006